File: Modul 5/Soal 3/PRAK503-NIM-Nama.c
#include <stdio.h>

int maks(int a, int b){
    if(a > b){
        return a;
    }
    else{
        return b;
    }
}

int mins(int a, int b){
    if(a < b){
        return a;
    }
    else{
        return b;
    }
}

int main(){
    int batas = 0;
    int max = -100000;
    int min = 100000;
    int bilangan;

    scanf("%d", &bilangan);

    while(batas < bilangan){
        int nilai;

        scanf("%d", &nilai);

        max = maks(max, nilai);
        min = mins(min, nilai);

        batas++;
    }
    printf("%d %d", max, min);
}

File: Modul 5/Soal 3/PRAK503-NIM-Nama.py
def maks(a, b):
    if(a > b):
        return a
    else:
        return b
    
def mins(a, b):
    if(a < b):
        return a
    else:
        return b
    
batas = []
max = -100000
min = 100000

bilangan = int(input())

while len(batas) < bilangan:
    nilai = list(map(int, input().split()))
    batas.extend(nilai)

for i in range (len(batas)):
    max = maks(batas[i-1], max)
    min = mins(batas[i-1], min)

print(max, min)